### PR TITLE
Alphabetised Legislation Listings page

### PR DESCRIPTION
**Pull request by roger-cts**
Creates inline separated alphabetical listings which then leads to second page with html list of all node titles starting with that letter.

### CODE REVIEW

**Comment by longhotsummer** [comment]
Great. Please can you make it so that the works are listed under each letter:

image


These french translations are annoying!

**Comment by longhotsummer** [comment]
And please delete the old legislation listing view.

File: config/default/views.view.legislation_listing.yml [hunk 1]
```diff
@@ -0,0 +1,602 @@
...
+          id: area
+          table: views
+          field: area
+          relationship: none
+          group_type: group
+          admin_label: ''
+          empty: false
+          tokenize: false
+          content:
+            value: "<a href=\"../legislation\">Namibia Consolidated Acts Index Home Page</a>\r\n\r\nThis collection of consolidated legislation was initially prepared by the Namibian Government. <a href=\"https://laws.parliament.na/about-us/\"> Find out more here. </a> </br>\r\n\r\nThe pre- and post-independence annotated laws as well as the post-independence regulations were prepared by the Legal Assistance Centre in Namibia. </br>\r\n\r\nAmendments post November 2017 are applied on a continuous basis by the <a href=\"https://laws.africa/\"> Laws.Africa team and contributors. </a> </br>"
```
Comment (longhotsummer) [inline config/default/views.view.legislation_listing.yml:262]: This site is for all LIIs, so we can't have this Namibia-specific header here. Let's just remove it completely for now.

Comment (longhotsummer) [reply config/default/views.view.legislation_listing.yml:262]: Although I don't see it showing up in my view, so 🤷

**Comment by longhotsummer** [comment]
The view is adding a `row` class to the view-content div, which makes Bootstrap think the children are columns, so it stacks sideways. I'm not sure how we fix this.

**Comment by longhotsummer** [comment]
> The view is adding a `row` class to the view-content div, which makes Bootstrap think the children are columns, so it stacks sideways. I'm not sure how we fix this.

We may need to override a template somewhere :) or is there a way we can change that class?

liiweb 2020-05-15 13-59-02

**Comment by roger-cts** [comment]
> The view is adding a `row` class to the view-content div, which makes Bootstrap think the children are columns, so it stacks sideways. I'm not sure how we fix this.

I can remove that in the view setup I believe.

File: config/default/views.view.legislation_listing.yml (hunk 1 above)
Comment (roger-cts) [reply config/default/views.view.legislation_listing.yml:262]: Cool will remove that.

**Pull request closed without merging**